tests/BcMathCalculatorTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * This file is part of the Money library
+ *
+ * Copyright (c) 2011-2013 Mathias Verraes
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Money\Tests;
+
+use Money\Calculator\BcMathCalculator;
+use Money\Money;
+use PHPUnit_Framework_TestCase;
+
+class BcMathCalculatorTest extends PHPUnit_Framework_TestCase
```

Comment:
Can we create an abstract test case with an abstract `getCalculator` method to avoid duplication?